rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1726,6 +1726,34 @@ void should_CancelTask_when_CallingCancelEndpoint() {
     }
   }
 
+  @Nested
+  @TestInstance(Lifecycle.PER_CLASS)
+  class TerminateTasks {
+    @Test
+    void should_TerminateTask_when_CallingCancelEndpoint() {
```

Comment:
should_TerminateTask is enough

Reply:
will also adjust for `should_CancelTask`